.github/workflows/pr-labels.yml

```diff
@@ -1,9 +1,9 @@
 name: pr-labels
 
 on:
-  push:
-    branches:
-      - main
+  pull_request:
+    types:
+      - closed
```

Comment:
What would happen if I open a test PR that is not meant to be merged and just close it?
Should we check if the PR was closed by the open source bot?

Reply:
if the PR is closed but not merged, the commit will not correspond to a PR number (which is obtained by parsing the merged commit message) and should not output a merger or generate any response

Reply:
Okay, sounds good.